cometd-java/cometd-java-benchmark/cometd-java-benchmark-server/src/main/java/org/cometd/benchmark/server/CometDLoadServer.java

```diff
@@ -541,16 +549,17 @@ public long convert(long measure) {
         }
 
         private void reset() {
-            histogram.reset();
+            allHistograms.forEach(AbstractHistogram::reset);
         }
 
         private void updateLatencies(long begin, long end) {
-            histogram.recordValue(end - begin);
+            histogramTl.get().recordValue(end - begin);
         }
 
         private void print() {
+            Histogram histogram = allHistograms.stream().reduce(new Histogram(TimeUnit.MINUTES.toNanos(1), 3), (h1, h2)-> {h1.add(h2); return h1;});
```

Comment:
Split the lambda into multiple lines and brace at EOL.

Reply:
Ok.